.github/CODEOWNERS

```diff
@@ -21,6 +21,7 @@ riotfile.py                         @DataDog/apm-python
 .riot/requirements/                 @DataDog/apm-python
 CHANGELOG.md                        @DataDog/apm-python
 README.md                           @DataDog/apm-python
+tests/telemetry                     @DataDog/apm-python
```

Comment:
Or maybe guild? I think `apm-python` should be fine though, we can update this later if needed